docs/dev/README.md

```diff
@@ -1,80 +1,72 @@
 ---
-title: Working with Agent-based integrations
+title: Introduction to Agent-based Integrations
 kind: documentation
 ---
 
-## Overview
-
-Being able to see all of your metrics from across your infrastructure is key within Datadog. There are currently three ways to get data into Datadog:
-
-1. Push data from the source to Datadog
-2. Crawl the data source's API
-3. Have the Datadog Agent pick up the information from the source
-
-This guide covers the latter option, providing the information needed to work with the Agent based integrations, also referred to as Checks. Checks make the Agent able to collect and send metrics, events and service checks to Datadog.
-
-## Create a new integration
+## Why create an Integration?
 
 While we do have guides to submit [custom metrics][1] via our [API][2] and [code instrumentation][3], it's possible you might want to see a certain source become an integration available in the [official core repository][4] and bundled into the Agent package.
 
-Overall, the largest deciding factor in what integrations we build is what our clients request. You have two options to propose an integration:
+Overall, the largest deciding factor in what integrations we build is what our clients request. There are two ways to propose an integration:
 
 * [Reach out to support@datadoghq.com][5] and tell us what metrics you would like to see from a given source.
 * Implement the integration yourself and submit the code to the [official extras repository][6].
 
-If you want to create a new Check from scratch, start by looking at the [howto][10] documentation.
+## Development basics
 
-## Development guide
-
-Follow these instructions to get a working copy of any check on your local Python environment; this is mostly useful to run tests or for tinkering in general.
+The remainder of this document introduces you to the base knowledge and setup required to start working on your own Integration. Once you're comfortable with this content, you can move on to the [technical specifics of Integration development][10].
```

Comment:
r/`Once you're comfortable with this content, you can move` / `Once you're comfortable with this content, move `